src/Lecture1.hs

```diff
@@ -63,7 +64,11 @@ sumOfSquares x y = error "TODO!"
 
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit inputNum = 
+    if (inputNum < 0) 
+    then mod ((-1)*inputNum) 10
+    else mod inputNum 10
```

Comment:
That's correct! 💯 
You can simplify the implementation a bit using the standard function `abs` that returns an absolute value of a number:

```haskell
λ: abs 7
7
λ: abs (-7)
7
```

Reply:
Replace with standard `abs`. Thx!